test/functional/shared/resources/testdata/recipes/test-terraform-recipes/azure-storage/main.tf

```diff
@@ -7,21 +7,25 @@ terraform {
   }
 }
 
+resource "random_id" "unique_name" {
+  byte_length = 8
+}
+
 resource "azurerm_storage_account" "test_storage_account" {
-  name = var.name
+  name = "account-${random_id.unique_name.id}"
   resource_group_name = var.resource_group_name
   location = var.location
   account_tier = "Standard"
   account_replication_type = "LRS"
 }
 
 resource "azurerm_storage_container" "test_container" {
-  name = "test-container"
+  name = "container-${random_id.unique_name.id}"
   storage_account_name = azurerm_storage_account.test_storage_account.name
 }
 
 resource "azurerm_storage_blob" "test_blob" {
-  name = "test-blob"
```

Comment:
Good catch. I think it might be related to this or `test-container` above. I don't think it is due to another long-running test deleting it but a functional test that happens to be running at the same time might be deleting one of these.

Reply:
Interesting. Are we sharing resource groups between functional tests and long running tests?

The failure is due to `test-container` but I just decided to add uniqueness to all resource names.

Reply:
> Interesting. Are we sharing resource groups between functional tests and long running tests?

No, we are not actually. They all belong to a resource group, right? Can't seem to find out why this is caused for now.